test/init/init.test.js

```diff
@@ -26,8 +26,8 @@ const readFromPkgJSON = (path) => {
 const readFromWebpackConfig = (path) => readFileSync(join(path, 'webpack.config.js'), 'utf8');
 
 describe('init command', () => {
-    beforeAll(async () => {
-        await mkdir(rootAssetsPath);
+    beforeAll(() => {
+        if (!existsSync(rootAssetsPath)) mkdirSync(rootAssetsPath);
```

Comment:
Please prefer to use `if () {}`, bracers is good practice, we need improve eslint config for all webpack repos and using them across all repos

Reply:
yep on it

Reply:
done